src/hotspot/share/gc/shenandoah/shenandoahPhaseTimings.hpp

```diff
@@ -99,8 +99,8 @@ class outputStream;
   f(conc_rendezvous_roots,                          "Rendezvous")                      \
   f(conc_evac,                                      "Concurrent Evacuation")           \
                                                                                        \
-  f(init_update_refs_gross,                         "Pause Init  Update Refs (G)")     \
-  f(init_update_refs,                               "Pause Init  Update Refs (N)")     \
+  f(init_update_refs_gross,                         "Pause Init Update Refs (G)")      \
+  f(init_update_refs,                               "Pause Init Update Refs (N)")      \
```

Comment:
Looks like unrelated spacing changes

Reply:
Do you want me to revert this and put it in a separate PR? Maybe when we add a phase for remembered set scanning?

Reply:
You could propose it upstream. However, looking at it again, it may have been done on purpose, to line up "Init Update Refs" with "Final Update Refs". Not sure how reasonable that is, though. I'd revert that part of the change here, though.